src/app/loaders/express/session.ts

```diff
@@ -6,14 +6,19 @@ import { Connection } from 'mongoose'
 
 import config from '../../config/config'
 
+export const ADMIN_LOGIN_SESSION_COOKIE_NAME = config.isDev
+  ? 'formsg.connect.sid'
```

Comment:
does it work on non-production devs too? (excluding `local`)

Reply:
Yup, `config.isDev` is true for local dev and staging environments

from [config.ts](https://github.com/opengovsg/FormSG/blob/develop/src/app/config/config.ts#L41-L43):

```
const isDev =
  basicVars.core.nodeEnv === Environment.Dev ||
  basicVars.core.nodeEnv === Environment.Test
```